code/modules/fields/timestop.dm

```diff
@@ -27,6 +27,9 @@
 	for(var/mob/living/L in GLOB.player_list)
 		if(locate(/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop) in L.mind.spell_list) //People who can stop time are immune to its effects
 			immune[L] = TRUE
+	for(var/mob/living/simple_animal/hostile/guardian/G in GLOB.parasites)
+		if(locate(/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop) in var/mob/living/summoner.mind.spell_list) //It would only make sense that a person's stand would also be immune.

```

Comment:
first of all we need to check if our summoner even exists
`if(G.summoner && G.summoner.mind && locate(/obj/effect/proc_holder/spell/aoe_turf/conjure/timestop) in G.summoner.mind.spell_list)`

Reply:
Thanks, I just realized this about 30 mins after you pointed it out.